3.5
An error occurred while executing the following cell:
------------------
import warnings
warnings.filterwarnings('ignore')
# Display the first lines of the searches.csv file to identify the key column parameters and delimiter
from itertools import islice
with open("searches.csv") as f:
    print(list(islice(f, 3)))
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mFileNotFoundError[0m                         Traceback (most recent call last)
[0;32m<ipython-input-1-1ccb25dc5e29>[0m in [0;36m<module>[0;34m()[0m
[1;32m      3[0m [0;31m# Display the first lines of the searches.csv file to identify the key column parameters and delimiter[0m[0;34m[0m[0;34m[0m[0m
[1;32m      4[0m [0;32mfrom[0m [0mitertools[0m [0;32mimport[0m [0mislice[0m[0;34m[0m[0m
[0;32m----> 5[0;31m [0;32mwith[0m [0mopen[0m[0;34m([0m[0;34m"searches.csv"[0m[0;34m)[0m [0;32mas[0m [0mf[0m[0;34m:[0m[0;34m[0m[0m
[0m[1;32m      6[0m     [0mprint[0m[0;34m([0m[0mlist[0m[0;34m([0m[0mislice[0m[0;34m([0m[0mf[0m[0;34m,[0m [0;36m3[0m[0;34m)[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0m

[0;31mFileNotFoundError[0m: [Errno 2] No such file or directory: 'searches.csv'
FileNotFoundError: [Errno 2] No such file or directory: 'searches.csv'

Executability                            : False
